tests/unit/handlers/taskUpdateHandler.test.ts

```diff
@@ -50,4 +51,22 @@ describe("sendTaskUpdatesHandler", () => {
     expect(result.status).toBe(401);
     expect(jsonResponse).toEqual(response.UNAUTHORIZED);
   });
+  it("should return success response if task update is sent successfully", async () => {
+    const mockRequest = generateDummyRequestObject({
+      url: "/task/update",
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: "Bearer dummyToken",
+      },
+    });
+    mockRequest.json = jest.fn().mockResolvedValue(mockData);
+    const result: JSONResponse = await sendTaskUpdatesHandler(
+      mockRequest,
+      mockEnv
+    );
+    expect(result.status).toBe(200);
+    const res: JSONResponse = await result.json();
+    expect(res).toBe("Task update sent on Discord's tracking-updates channel.");
```

Comment:
- Suggestion:
Keep the `Task update sent...` in a common constant file so that if we want to change the message it can be done from a single place.

Reply:
Thankyou for your suggestion, the changes has been made please check.